src/hub_predtimechart/hub_config.py

```diff
@@ -136,13 +139,12 @@ def get_sorted_values_or_first_config_ref_date(reference_dates: set[str]):
                 return sorted(list(reference_dates))
 
 
-        # loop over every (reference_date X model_id) combination.
+        # loop over every (reference_date X model_id) combination
         as_ofs = {self.fetch_target_id: set()}
         for reference_date in self.reference_dates:  # ex: ['2022-10-22', '2022-10-29', ...]
             for model_id in self.model_id_to_metadata:  # ex: ['Flusight-baseline', 'MOBS-GLEAM_FLUH', ...]
                 model_output_file = self.model_output_file_for_ref_date(model_id, reference_date)
                 if model_output_file:
-                    # todo xx extract to function, call from here and _generate_json_files()
```

Comment:
My vague recollection was that there was similar logic to this in some other place.  If that's right, it might be worth keeping track of this opportunity for refactoring?  We could file an issue to address it later if you think it'd be worthwhile.

Reply:
I looked at creating a function shared by both locations, but I decided there was enough differences to not justify it. A judgement call...